packages/web-pkg/src/components/sideBar/Spaces/Details/SpaceDetailsMultiple.vue

```diff
@@ -1,21 +1,121 @@
 <template>
-  <div class="oc-mt-xl">
-    <div class="oc-flex space-info">
-      <oc-icon name="layout-grid" size="xxlarge" />
-      <p v-translate>Multiple spaces selected</p>
+  <div id="oc-spaces-details-multiple-sidebar">
+    <div class="spaces-preview oc-mb">
+      <div class="spaces-preview-body">
+        <oc-icon class="preview-icon" size="xxlarge" variation="passive" name="layout-grid" />
+        <p class="preview-text" data-testid="selectedFilesText" v-text="selectedSpacesString" />
```

Comment:
Let's either remove `data-testid` (as we don't use it in the tests), or rename it to `selectedSpacesText`. The same goes for the testids down below.